tests/components/aurora_abb_powerone/test_sensor.py

```diff
@@ -123,21 +126,28 @@ async def test_sensor_dark(hass: HomeAssistant) -> None:
         power = hass.states.get("sensor.mydevicename_power_output")
         assert power is not None
         assert power.state == "45.7"
+    utcnow = dt_util.utcnow()
 
     # sunset
     with patch("aurorapy.client.AuroraSerialClient.connect", return_value=None), patch(
         "aurorapy.client.AuroraSerialClient.measure",
         side_effect=AuroraTimeoutError("No response after 10 seconds"),
+    ), patch(
+        "aurorapy.client.AuroraSerialClient.cumulated_energy",
+        side_effect=AuroraTimeoutError("No response after 3 tries"),
     ):
-        async_fire_time_changed(hass, utcnow + timedelta(seconds=60))
+        async_fire_time_changed(hass, utcnow + SCAN_INTERVAL * 2)
```

Comment:
Please use the freezer fixture for changing time

Reply:
Done [c36f1f5](https://github.com/home-assistant/core/pull/72363/commits/c36f1f5dfdc6cb49fa062b370e398bef066f7755)